### DISCUSSION TITLE
Plots in your thesis?

### DISCUSSION THREAD

**Discussion by suckingalemon** [Submanifold/latex-mimosis#54] [Q&A] [answered]
Just wondering how you created the plots in your thesis here:

[Persistent Homology in Multivariate Data Visualization.pdf](https://github.com/user-attachments/files/16783031/Persistent.Homology.in.Multivariate.Data.Visualization.pdf)

They fit so well within your LaTeX document. I would love to know how you achieved that.

Thank you.

**Comment by Pseudomanifold** [comment, marked as answer]
Thanks! This was achieved mainly with a combination of Inkscape and TikZ.

**Reply by suckingalemon** [reply]
Thank you. Incredibly useful template, by the way.

**Reply by Pseudomanifold** [reply]
Thanks a lot! BTW: I can make the source code of the figures available to you if you need them. I just ask for an acknowledgement or a corresponding citation in the figure caption.
